### DISCUSSION TITLE
detect.py code

### DISCUSSION THREAD

**Discussion by michaelbs09** [ultralytics/yolov5#4714] [Q&A] [answered]
I'm trying to find the variable in the detect.py file where the detected class is saved as a string. Can someone help me?

**Comment by glenn-jocher** [comment, marked as answer]
@michaelbs09 this line prints the detection to screen. `n` is the number of detections per class, `names` is the class list, and `c` is the class index:
https://github.com/ultralytics/yolov5/blob/25a7e1dae59e75d2c401a49cd2c7b76a7cf07139/detect.py#L192-L196

**Reply by michaelbs09** [reply]
Thank you very much @glenn-jocher , I was in doubt.
